param/coercer/coercer_test.go

```diff
@@ -8,21 +8,47 @@ import (
 )
 
 func TestCoerceParams_AnyOfCoercion(t *testing.T) {
-	schema := &spec.Schema{Properties: map[string]*spec.Schema{
-		"objectorintkey": {
-			AnyOf: []*spec.Schema{
-				{Type: objectType},
-				{Type: integerType},
+	// `anyOf` with basic types
+	{
+		schema := &spec.Schema{Properties: map[string]*spec.Schema{
+			"bool_or_int_key": {
+				AnyOf: []*spec.Schema{
+					{Type: objectType},
```

Comment:
minor but why is the key `bool_or_int_key` but the options are object or int?

Reply:
Ah, good eyes Remi! Thanks. I just added a fix in #98, which also uncovered a different problem, so it patches that as well.